src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyCost = 40;
+  let totalCost = dailyCost * days;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
```

Comment:
Similarly, define a constant for the short-term discount condition and amount, such as `SHORT_TERM_RENTAL_DAYS` and `SHORT_TERM_RENTAL_DISCOUNT`.